packages/vue/src/components/basic/SelectedFilters.jsx

```diff
@@ -148,7 +148,7 @@ const SelectedFilters = {
 							component,
 							`${component}-${index + 1}`,
 							() => this.remove(component, value),
-							`${selectedValues[component].label}: ${valueToRender}`,
+							`${selectedValues[component].label}: ${decodeHtml(valueToRender)}`,
```

Comment:
Why do you need to use decode html in selected filters?
The selected value should be decoded.

Reply:
https://www.loom.com/share/ab112bf4045a48dcb05fb63c71720890